### PR TITLE
QF-2112-Refresh_standards_inner_api

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

File: src/SFA.DAS.RequestApprenticeTraining.Database/Tables/Standard.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿CREATE TABLE [dbo].[Standard]
 (
-    [StandardReference] VARCHAR(25) NOT NULL PRIMARY KEY, 
-    [StandardTitle] VARCHAR(100) NOT NULL,
+    [StandardReference] NVARCHAR(6) NOT NULL PRIMARY KEY,
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Database/Tables/Standard.sql]: The change from 25 to 6 is understandable but why is a 'n' varchar needed - what not ascii character would be expected here? - or has that just been used in the Courses API? so you are following that through?

File: src/SFA.DAS.RequestApprenticeTraining.Application.UnitTests/Commands/SubmitProviderResponse/WhenValidatingSubmitProviderResponseCommand.cs [hunk 2]
```diff
@@ -1,108 +0,0 @@
-﻿using FluentValidation.TestHelper;
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Application.UnitTests/Commands/SubmitProviderResponse/WhenValidatingSubmitProviderResponseCommand.cs]: Why is this being removed?

Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Application.UnitTests/Commands/SubmitProviderResponse/WhenValidatingSubmitProviderResponseCommand.cs]: removed by mistake.  I have re-added it

File: src/SFA.DAS.RequestApprenticeTraining.Database/Tables/Standard.sql (hunk 1 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Database/Tables/Standard.sql:3]: Changed to nvarchar(6) to match EmployerRequest.StandardReference, but its just a varchar in the Courses DB.  Should I change them both to varchar(6)

Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Database/Tables/Standard.sql]: Changed Standard.StandardReference and EmployerRequest.StandardReference to varchar(6)

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**